src/nwbuild.js

```diff
@@ -72,10 +72,18 @@ export const nwbuild = async (options) => {
     }
 
     // Parse options, set required values to undefined and flags with default values unless specified by user
-    options = await parse(options);
+    options = await parse(options, nwPkg);
+
+    // Variable to store nwDir file path
+    nwDir = `${options.cacheDir}/nwjs${
+      options.flavour === "sdk" ? "-sdk" : ""
+    }-v${options.version}-${options.platform}-${options.arch}`;
```

Comment:
This feels like it's trying to do too much in a compact space. I tried breaking it up on multiple lines. Feel free to ignore/resolve this comment.
```suggestion
    nwDir = [
      options.cacheDir,
      "/nwjs",
      options.flavour === "sdk" ? "-sdk" : "",
      "-v" + options.version,
      "-" + options.platform,
      "-" + options.arch
    ].join('');
```

Reply:
```js
    const flavorString = options.flavour === "sdk" ? "-sdk" : "";
    nwDir = `${options.cacheDir}/nwjs${flavorString}-v${options.version}-${options.platform}-${options.arch}`;
```
IMO, this is more readable than Jared's solution. 

I also notice you aren't using `node:path` everywhere in this project and really should be.

Reply:
I think this works for now as long as the manifest format is not changed.